config/jest-preprocess-typescript.js

```diff
@@ -0,0 +1,11 @@
+const tsc = require("typescript");
+const tsConfig = require("../tsconfig.json");
+
+module.exports = {
+  process(src, path) {
+    if (path.endsWith(".ts")) {
```

Comment:
i might use, path.extname but i think it will be the same

https://nodejs.org/dist/latest-v8.x/docs/api/path.html#path_path_extname_path

Reply:
Oh I like that! The official Jest-Typescript example did this. 🤦‍♂️